### DISCUSSION TITLE
Allow to set "Start" tag + allow removal of "All" tag

### DISCUSSION THREAD

**Discussion by mathias723** [linuxserver/Heimdall#1307] [Ideas]
Hi, 
I love the option to use tags as tags.
If possible, please allow to set a starting tag. Currently youre starting with "all". It would be great to allow to have specific tag when the page is opened and maybe even to have an option to hide the" al"l tag in the selection.

Thank you.

**Comment by Neclony** [comment]
After setting the option "Treat Tags As: Tag" at least 2 other options would be awesome.
Hide "All" Tag
Set Default Tag (this would be one from the list of user created tags, instead of the all tag)

Also the "Home Dashboard" tag should appear in the menu "tags list" for users to edit or delete if not needed

**Reply by Quaesitio** [reply]
Yes, that is a very good suggestion. I am really forward for this feature to be implemented.

**Comment by SBerg1980** [comment]
Yes me to +1

**Comment by david-livingstone-wpp** [comment]
+1 for me.

I accomplished this by adding custom css and javascript via settings->advanced.  YMMV.

Custom CSS:
div[data-tag="all"] {
    display:none;
}

Custom JavaScript:
const div = [...document.querySelectorAll('#taglist div')]
  .find(el => el.textContent.includes('Home dashboard'));

if (div) {
  window.addEventListener('load', () => {
    setTimeout(() => {
        div.click();
    }, 100); // 1000ms = 1 second
});
}

**Comment by lloydbayley** [comment]
God almighty, I can't remember exactly why I subscribed to this thread after a year but I'm jolly-glad I did. Now I have the chance to learn about some CSS and Java! However, I must admit that I had to look up YMMV. I'm not sure whether to be annoyed about getting older of the invention of these acronyms :)
